actors/miner/src/lib.rs

```diff
@@ -1950,6 +1950,70 @@ impl Actor {
         Ok(ProveCommitSectors3Return { activation_results: result })
     }
 
+    fn confirm_sector_proofs_valid(
+        rt: &impl Runtime,
+        params: ConfirmSectorProofsParams,
+    ) -> Result<(), ActorError> {
+        rt.validate_immediate_caller_is(std::iter::once(&STORAGE_POWER_ACTOR_ADDR))?;
+
+        /* validate params */
+        // This should be enforced by the power actor. We log here just in case
+        // something goes wrong.
+        if params.sectors.len() > ext::power::MAX_MINER_PROVE_COMMITS_PER_EPOCH {
```

Comment:
Remove this check.  We should clean this up to the minimal thing that lotus seed can use.